### ISSUE TITLE
Make all PRs on GitHub

### ISSUE THREAD

**Issue by bobturneruk** [RSE-Sheffield/git-github-zero-to-hero#86]
I suggest, for consistency we should make all of the PRs in all exercises using GitHub, not GitKraken. We already do this for evolottery.

**Comment by ns-rse** [comment]
I agree, its perhaps habit as I've not (yet) worked out how to do them in Emacs (although it is possible, I have all issues and PRs listed :wink: ) but I think because people may go away and use another IDE its more useful to know a universal way of creating and merging PRs and whilst the current material does show both GitKraken and GitHub reinforcing the process on GitHub would be useful and slightly less confusing demonstrating different methods.
